pallet/src/balance.rs

```diff
@@ -69,14 +69,38 @@ impl<T: Config + SysConfig> BalanceAdapter<T> {
         }
     }
 
+    /// Creates a new [`BalanceAdapter`] with unlimited cheque-limits for every signer for the
+    /// usage within a dry-run. Therefor, the extracted arguments and the signer count of a given
+    /// script transaction has to be provided.
+    pub fn for_dry_run(args: &[&[u8]], signer_count: usize) -> Result<BalanceAdapter<T>, Error<T>> {
+        use crate::log;
```

Comment:
nit: could be more consistent to declare it with the rest of the use statements above?